### ISSUE TITLE
pkgdown site doesn't link to github repo

### ISSUE THREAD

**Issue by apreshill** [tidymodels/rsample#120]
I think this [line](https://github.com/tidymodels/rsample/blob/b1f55d9df9c9632f5310b044e8e0594703ef2d63/DESCRIPTION#L34):

```
URL: https://tidymodels.github.io/rsample
```

needs to be:

```
URL: https://tidymodels.github.io/rsample, https://github.com/tidymodels/rsample
```

See the [pkgdown `DESCRIPTION` file](https://github.com/r-lib/pkgdown/blob/134033d9d744c548c6fbe297bc0fbd1a2f498e83/DESCRIPTION#L17) as an example.

@topepo @DavisVaughan good one for tidydevday label?

**Comment by DavisVaughan** [comment]
You will also need to add a `url` field to the `_pkgdown.yml`. The complete set of steps is documented nicely in this vignette https://pkgdown.r-lib.org/articles/linking.html

**Comment by danidata** [comment]
Hello all :)

It looks like the URL was changed in the description file here: https://github.com/tidymodels/rsample/blob/8868fc501cd594141c2634184a814f17f87d42bb/DESCRIPTION#L34

and in the `_pkgdown.yml` here: https://github.com/danidata/rsample/blob/8868fc501cd594141c2634184a814f17f87d42bb/_pkgdown.yml#L1

Have I missed the request here? Thanks!

**Comment by DavisVaughan** [comment]
This was closed by #125 sorry!

**Comment by danidata** [comment]
No worries!
